packages/cli/internal/pkg/cli/account_activate.go

```diff
@@ -33,10 +35,11 @@ A new VPC will be created if not specified.`
 	accountTagsDescription = `A list of comma separated tags to be applied to all AGC resources in this account
 (i.e. --tags "k1=v1","k2=v2"). Each key-value pair must be quoted as shown in the example,
 otherwise the parsing will fail.`
-	cdkCoreDir   = ".agc/cdk/apps/core"
-	bucketPrefix = "agc"
-	activateKey  = "activate"
-	bootstrapKey = "bootstrap"
+	cdkCoreDir    = ".agc/cdk/apps/core"
+	bucketPrefix  = "agc"
+	activateKey   = "activate"
+	bootstrapKey  = "bootstrap"
+	coreStackName = "Agc-Core"
```

Comment:
Can you move this to [awsresources.go](https://github.com/aws/amazon-genomics-cli/blob/main/packages/cli/internal/pkg/cli/awsresources/awsresources.go) as a `RenderCoreStackName()` function?